### ISSUE TITLE
Missing icons in version 4

### ISSUE THREAD

**Issue by catherinek** [joypixels/emojione#617]
Hello there, the following icons are missing from the Version 4 Emoji JSON.
:tm: :copyright: :registered:
`:tm: :copyright: :registered:`

They also don't get translated when using the PHP library.
Please help.

**Comment by brenc** [comment]
Noticed that here too.

**Comment by sampaiodiego** [comment]
all the following emojis are missing:

`:tm:`
`:copyright:`
`:registered:`
`:digit_zero:`
`:digit_one:`
`:digit_two:`
`:digit_three:`
`:digit_four:`
`:digit_five:`
`:digit_six:`
`:digit_seven:`
`:digit_eight:`
`:digit_nine:`
`:pound_symbol:`
`:asterisk_symbol:`

**Comment by brenc** [comment]
Yeah I don't get the lack of response on this. I can't upgrade because of this.

**Comment by caseyahenson** [comment]
Hi all! Thanks for your patience as we've been a bit short staffed as of late.

This issue is partially resolved (still seeing that digits are not replacing) over at the new emoji-toolkit repository: https://github.com/joypixels/emoji-toolkit. I'll be sure this is fully resolved in the next few days. We'll gradually be transitioning to that repository and eventually be deprecating this repository (well into the future). We recommend switching your integrations to emoji-toolkit -- please see emoji-toolkit/UPGRADE.md before switching.

Official announcement and further details coming soon :)

**Comment by dbrgn** [comment]
@caseyahenson thanks for the update! If this is resolved for the non-digits, shouldn't the JSON for the copyright symbol...

```js
// emoji.json
{
    "name": "copyright",
    "unicode_version": 1.1,
    "category": "symbols",
    "order": 199,
    "display": 1,
    "shortname": ":copyright:",
    "shortname_alternates": [

    ],
    "ascii": [

    ],
    "diversity": null,
    "diversities": [

    ],
    "gender": null,
    "genders": [

    ],
    "code_points": {
        "base": "00a9",
        "fully_qualified": "00a9",
        "non_fully_qualified": "00a9",
        "output": "00a9",
        "default_matches": [
            "00a9-fe0f"
        ],
        "greedy_matches": [
            "00a9-fe0f"
        ],
        "decimal": ""
    },
    "keywords": [
        "copyright"
    ]
}
```

...have the emoji variant selector in `fully_qualified`?

**Comment by brenc** [comment]
I switched everything over to `emoji-toolkit` and `emoji-assets` which worked for me. Perhaps the old repos should be marked as deprecated?

**Comment by caseyahenson** [comment]
This is fixed in the 5.0 release https://github.com/joypixels/emoji-toolkit/releases/tag/5.0.0.
